package.json

```diff
@@ -35,7 +36,7 @@
     "tslint-config-prettier": "^1.18.0",
     "typescript": "4.5.5"
   },
-  "homepage": "https://github.com/benwinding/react-admin-firebase",
+  "homepage": "https://github.com/rinfimate/react-admin-firebase",
```

Comment:
none of the changes in this file besides the dependency should be part of the pull request

Reply:
You might need to make another branch on your fork say `rinfimate:add-multi-tenant-support` in order to only merge in changes that are necessary (_you then can update this PR with the new branch_)... otherwise you'll need to change your `master` branch to remove these changes